beets/util/artresizer.py

```diff
@@ -77,7 +77,13 @@ def pil_resize(maxwidth, path_in, path_out=None, quality=0):
         im = Image.open(util.syspath(path_in))
         size = maxwidth, maxwidth
         im.thumbnail(size, Image.ANTIALIAS)
-        im.save(util.py3_path(path_out), quality=quality)
+
+        if quality == 0:
+            # Use PIL's default quality.
+            im.save(util.py3_path(path_out))
```

Comment:
I tried passing a `None` quality instead of having two `save` calls, but that causes PIL to blow up:

```Text
Traceback (most recent call last):
  File "./beet", line 23, in <module>
    beets.ui.main()
  File "<dir>/beets/ui/__init__.py", line 1269, in main
    _raw_main(args)
  File "<dir>/beets/ui/__init__.py", line 1256, in _raw_main
    subcommand.func(lib, suboptions, subargs)
  File "<dir>/beetsplug/fetchart.py", line 1036, in func
    self.batch_fetch_art(lib, lib.albums(ui.decargs(args)), opts.force,
  File "<dir>/beetsplug/fetchart.py", line 1096, in batch_fetch_art
    candidate = self.art_for_album(album, local_paths)
  File "<dir>/beetsplug/fetchart.py", line 1076, in art_for_album
    out.resize(self)
  File "<dir>/beetsplug/fetchart.py", line 139, in resize
    self.path = ArtResizer.shared.resize(plugin.maxwidth, self.path,
  File "<dir>/beets/util/artresizer.py", line 215, in resize
    return func(maxwidth, path_in, path_out, quality=quality)
  File "<dir>/beets/util/artresizer.py", line 84, in pil_resize
    im.save(util.py3_path(path_out), quality=quality)
  File "<dir>/venv/lib/python3.8/site-packages/PIL/Image.py", line 2151, in save
    save_handler(self, fp, filename)
  File "<dir>/venv/lib/python3.8/site-packages/PIL/JpegImagePlugin.py", line 633, in _save
    raise ValueError("Invalid quality setting")
ValueError: Invalid quality setting
```

Reply:
Thanks for trying that out!! Just for fun, I dug into the PIL source a little bit, and it seems like the default value might be `-1`:
https://github.com/python-pillow/Pillow/blob/1cb3e2f742f70cb233024b6baa11358e78c73101/src/PIL/JpegImagePlugin.py#L619

Reply:
Looks like this works, thanks! 🎉